### ISSUE TITLE
Posté audio

### ISSUE THREAD

**Issue by rod93** [Kodi-vStream/venom-xbmc-addons#853]
Bonjour
Est il normal de ne pas pouvoir sélectionner les pistes audio ?

**Comment by TmpName** [comment]
Oui, tres peu d'hebergeurs le permettent, de memoire seul uptobox/uptostream.
